python/trainer_configurations.yaml

```diff
@@ -1,27 +1,40 @@
 default:
-    batch_size: 256
-    beta: 2.5e-3
-    buffer_size: 5000
+    batch_size: 32
+    beta: 5.0e-3
```

Comment:
Would prefer adding a single line comment articulating the change in default parameters. Perhaps referencing a trial run?